automan/tests/test_cluster_manager.py

```diff
@@ -187,6 +187,34 @@ def test_remote_bootstrap_and_sync(self):
         dest = os.path.join(self.root, 'automan', project_name, 'script.py')
         self.assertTrue(os.path.exists(dest))
 
+    @mock.patch.object(ClusterManager, '_bootstrap')
+    @mock.patch.object(ClusterManager, '_delete_outputs')
+    def test_delete_outputs_from_hosts(self, mock_delete_outputs, mock_bootstrap):
+        # Given
+        cm = ClusterManager()
+        cm.add_worker('host', home='/home/foo', nfs=False)
+        cm.add_worker('host1', home='/home/foo1', nfs=False)
+
+        # When
+        cm.delete('outputs', ['host', 'host1'])
+
+        # Then
+        mock_delete_outputs.assert_called_with('host1', '/home/foo1', 'outputs')
```

Comment:
Why is this only called once?  Shouldn't it be called with host and host1?

Reply:
fixed.